src/controllers/userController/updateResetPassword.js

```diff
@@ -0,0 +1,35 @@
+import userServices from '../../services/user.service';
+import { verifyToken} from '../../utils/auth';
+import AuthorizationError from '../../utils/Errors/authorizationError';
+import BadRequestError from '../../utils/Errors/badRequestError';
+import ApplicationError from '../../utils/Errors/applicationError';
+import NotFoundRequestError from '../../utils/Errors/notFoundRequestError';
+let usedToken ='';
+
+const verifyResetPassword = async (req, res, next) => {
+  try {
+    const { token } = req.query;
+    const { password, confirmPassword } = req.body;
+
+    if(usedToken === token){throw new ApplicationError("Can not reset the password again!")}
```

Comment:
but used token is initialized with empty string , so this have to come from table maybe